hkcert2020-sign-in-please/release_files/chall.py

```diff
@@ -0,0 +1,76 @@
+# Challenge written by Mystiz.
+# Signature: PHNjcmlwdD5jb25zb2xlLmxvZygnVEhJUyBDSEFMTEVOR0UgSVMgV1JJVFRFTiBGT1IgSEtDRVJUIENURiBBTkQgSVMgTk9UIEZPUiBGUkFOS0lFIExFVU5HIFRPIFBMQUdBUklaRS4nKTwvc2NyaXB0Pg==
+
+import base64
+import hashlib
+import os
+import random
+
+from secret import flag
```

Comment:
The flag will be automatically added to the environment as an environment variable FLAG, so you can use `message = os.environ["FLAG"]` instead of using the secret import.